src/request/request-tab.js

```diff
@@ -43,3 +44,10 @@ module.exports.registerTab({
     title: 'Request',
     component: request.Request
 });
+
+module.exports.registerTab({
+    key: 'tab.logging',
+    title: 'Trace',
+    component: LoggingComponent,
+    viewModel: LoggingViewModel
```

Comment:
Yeah, it's still a little messy.  I'll move things around to be a little more clear.